activerecord/lib/active_record/core.rb

```diff
@@ -534,17 +533,18 @@ def pretty_print(pp)
       return super if custom_inspect_method_defined?
       pp.object_address_group(self) do
         if defined?(@attributes) && @attributes
-          column_names = self.class.column_names.select { |name| has_attribute?(name) || new_record? }
```

Comment:
I think `column_names` here was left over from 8ab9daf2803504edbd49f9c97a7f2eb484f15747 so changed to `attribute_names`.